src/core_atmosphere/physics/physics_wrf/module_mynnsfclay.F90

```diff
@@ -385,7 +386,7 @@ subroutine mynnsfclay(                                 &
     ! according to Akb(1976), Eq(12).
     !--------------------------------------------------------
     br(i)=govrth(i)*za(i)*dthvdz/(wspd(i)*wspd(i))
-    if (itimestep == 1) then
+    if (.not. restart .and. itimestep == 1) then
```

Comment:
Do we want to add `.not. cycling` into line 389?

Reply:
@guoqing-noaa yes, it is confusing, but there is a line in the module_mynnsfclay_driver.F90
 if(restart .or. cycling)restart_or_cycle=.true.
and restart_or_cycle is passed into  module_mynnsfclay.F90.

Reply:
Master @guoqing-noaa has a godo point. Perhaps we should change "restart" to "restart_or_cycle" in this module too.